include/deal.II/numerics/vector_tools.h

```diff
@@ -924,6 +933,29 @@ namespace VectorTools
                 VectorType &vec_result);
 
   /**
+   * Generic implementation for the project() function with FiniteElements that
+   * are supported by the MatrixFree class.
+   * Use this function if you use more than four components or the degree of
+   * your FiniteElement is higher than eight. You have to specify the first
+   * template arguments yourself.
```

Comment:
Can you explain why this function should only be used if you have MORE components and a HIGHER polynomial degree?